cmd/cleanup/cleanup.go

```diff
@@ -23,6 +24,8 @@ func NewCmd(cConfig *cmd.CleanupConfiguration, loaders []cli.ResourceLoader) *cl
 }
 
 func cleanupCommand(cConfig *cmd.CleanupConfiguration) error {
+	ctx := context.Background()
```

Comment:
What do you think about using a context with a signal?

```suggestion
	ctx := cmd.ContextWithSignal(context.Background())
```